.snyk

```diff
@@ -1,10 +1,54 @@
-# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities. -*- mode: yaml -*-
 version: v1.13.5
+language-settings: 
+  python: "2.7"
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
   SNYK-PYTHON-BEAKER-575115:
     - '*':
         reason: >-
-          No remediation available yet; Not affecting us since the storage is not accessible to any other client
-        expires: 2021-01-10T06:00:00.000Z
-patch: {}
\ No newline at end of file
+          No remediation available yet; Not affecting us 
+          since the storage is not accessible to any other client
+        expires: 2020-11-19T06:00:00.000Z
+  
+  SNYK-PYTHON-PYYAML-590151:
+    - '*':
+        reason: >-
+          No remediation available yet. YAML files are loaded only
+          as text fixtures (trusted source and execution context.)
+          Risk is acceptable.
+        expires: 2020-11-19T06:00:00.000Z
+
+  SNYK-PYTHON-SQLALCHEMY-173678:
+    - '*':
+        reason: >-
+          CKAN 2.8 doesn't appear to be compatible with SQLAlchemy 1.2.
+          CKAN 2.9 is, however.  To properly evaluate the risk,
+          we should examine all use of 'group_by' and 'order_by'
```

Comment:
@jbrown-xentity performed this analysis in CKAN 2.3 and no fixes were necessary. Maybe you can check with him on how to validate for 2.8.